conditionals.md

```diff
@@ -0,0 +1,227 @@
+
+```js
+10 < 12 // Evaluate to true
+```
+
+We can also use comparison operators on different data types like strings:
+
+```js
+'apples' === 'oranges' // false
+```
+ Since the two strings are not the same, the comparison statement evaluates to false. All comparison statements evaluate to either true or false and are made up of:
+
+- Two values that will be compared.
+- An operator that separates the values and compares them accordingly (>, <, <=,>=,===,!==).
+Example:
+
+```js
+let hungerLevel = 7;
+if (hungerLevel >= 7) {
+  console.log('Time to eat!');
+} else {
+  console.log('We can eat later!');
+}// output = Time to eat!
+```
+
+### Logical operators
+
+In conditionals, boolean `true` or `false` values will most times be in use. In javaScript *logical operators* is use to work with boolean value. There are three(3) logical operators:
+
+- The `and` operator (&&)
+- The `or` operator (||)
+- The `not` (!)
+
+When we use the `&&` operator, we are checking that two things are true:
+
+```js
+if (stopLight === 'green' && pedestrians === 0) {
+  console.log('Go!');
+} else {
+  console.log('Stop');
+}
+```
+
+When using the `&&` operator, both conditions must evaluate to true for the entire condition to evaluate to true and execute. Otherwise, `if` either condition is false, the `&&` condition will evaluate to false and the `else` block will execute.
+If we only care about either condition being true, we can use the `||` operator:
+
+```js
+if (day === 'Saturday' || day === 'Sunday') {
+  console.log('Enjoy the weekend!');
+} else {
+  console.log('Do some work.');
+}
+```
+
+When using the `||` operator, only one of the conditions must evaluate to true for the overall statement to evaluate to true.
+
+The `! not` operator reverses, or negates, the value of a boolean:
+
+```js
+let excited = true;
+console.log(!excited); // Prints false
+
+let sleepy = false;
+console.log(!sleepy); // Prints true
+```
+Essentially, the ! operator will either take a true value and pass back false, or it will take a false value and pass back true.
+
+Logical operators are often used in conditional statements to add another layer of logic to our code.
+
+### Truthy and Falsy
+
+`truthy` an `falsy` are used to evaluate a non-boolean condition, like string and numbers.
+
+Here’s an example:
+
+```js
+let myVariable = 'I Exist!';
+if (myVariable) {
+   console.log(myVariable)
+} else {
+   console.log('The variable does not exist.')
+}
+```
+
+The code block in the if statement will run because myVariable has a truthy value; even though the value of myVariable is not explicitly the value true, when used in a boolean or conditional context, it evaluates to true because it has been assigned a non-falsy value.
+
+Falsy values are:
+- 0
+- Empty strings like "" or ''
+- null which represent when there is no value at all
+- undefined which represent when a declared variable lacks a value
+- NaN, or Not a Number
```

Comment:
Put the values in code, for example, `0`, `""`, `''`. `null`, `undefined` and `NaN`.

Also `- NaN, or Not a Number` is misleading, `NaN` _means (not or)_ **Not a Number**.